src/Bicep.Core/TypeSystem/Az/OciTypeLoader.cs

```diff
@@ -7,22 +7,22 @@
 using System.IO.Compression;
 using Azure.Bicep.Types;
 
-namespace Bicep.Core.TypeSystem.Az
+namespace Bicep.Core.TypeSystem
 {
-    public class OciAzTypeLoader : TypeLoader
+    public class OciTypeLoader : TypeLoader
     {
         private readonly ImmutableDictionary<string, byte[]> typesCache;
-
-        public OciAzTypeLoader(ImmutableDictionary<string, byte[]> typesCache)
+        private const string typesArtifactFilename = "types.tgz";
+        public OciTypeLoader(ImmutableDictionary<string, byte[]> typesCache)
```

Comment:
[nit] Make constructor `private` if you want to enforce use of the `.FromTgz` method to instantiate.